source/includes/acts/reference.md

```diff
@@ -24,10 +24,10 @@ To begin using the Intelex API you will need:
 >Example Requests:
 
 ```CSharp 
-//JSON Body for Authentication Endpoint
+Body for Authentication Endpoint
 {"client_id":"Client ID Data",
 "client_secret":"Client Secret Data",
-"audience":"https://intelex_acts_api_url/api",
+"audience":"https://tenant.actsapi.intelex.com/v1/api",
```

Comment:
[tenant]

Reply:
@stevelavoie-pm  the above URL has been changed to https://[tenant].actsapi.intelex.com which is present in the latest commit - 3rd , Thanks .